appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobsProxy.java

```diff
@@ -79,24 +79,39 @@
     @Param(primary = true, optional = true)
     String jobName;
 
+    @Param(name = "offset", optional = true, defaultValue = "0")
+    String offSetValue;
```

Comment:
Make sure to add validation to these to prevent negative errors etc! May require adding to both the proxy and the main command. Will probably need a rework sometime in the future, but I would just validate both for now.